amplify/backend/function/fetchDomainBalance/src/api/graphql/schemas.js

```diff
@@ -89,6 +89,42 @@ module.exports = {
       }
     }
   `,
+  getAllDomainExpenditures: /* GraphQL */ `
```

Comment:
Maybe we should rename this to `getColonyExpenditures` or `getColonyDomainsExpenditures`